packages/fyndiq-component-alert/src/localStorage.js

```diff
@@ -0,0 +1,20 @@
+export const loadstate = () => {
+  try {
+    const serializedState = localStorage.getItem('count')
```

Comment:
One problem I see with that implementation is that if you have several alert components, they will all share the same localStorage variable `count`, and will interfere with each other. To fix that, the localstorage key name should maybe be passed as a prop to the Alert component.